src/interactivity/base/ServiceLocator.cpp

```diff
@@ -104,6 +104,24 @@ void ServiceLocator::RundownAndExit(const HRESULT hr)
 
 #pragma region Set Methods
 
+[[nodiscard]] NTSTATUS ServiceLocator::SetAccessibilityNotifier(_In_ std::unique_ptr<IAccessibilityNotifier>&& notifier)
+{
+    if (s_accessibilityNotifier)
+    {
+        NT_RETURN_NTSTATUS(STATUS_INVALID_HANDLE);
+    }
+    else if (!notifier)
+    {
+        NT_RETURN_NTSTATUS(STATUS_INVALID_PARAMETER);
+    }
+    else
+    {
+        s_accessibilityNotifier = std::move(notifier);
+    }
+
+    return STATUS_SUCCESS;

```

Comment:
I find it funny how each if/else branch does an early return, except for the branch that sets the value.😄

Reply:
I just copied the pattern man. Hahaha.